content/en/events/index.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
What it is this for?

Reply:
Ignorable infra stuff. This just tells Hugo that this whole tree is not to be rendered as normal pages. There's the same kind of file at the top of the boilerplates directory for the same reason.